db.go

```diff
@@ -246,6 +246,11 @@ type Handle struct {
 func (h Handle) WriteTxn(table TableMeta, tables ...TableMeta) WriteTxn {
 	db := h.db
 	allTables := append(tables, table)
+	for _, table := range allTables {
+		if table.tablePos() < 0 {
+			panic(tableError(table.Name(), ErrTableNotRegistered))
```

Comment:
Can we move this into the loop below that gathers the mutexes?

Reply:
Yes, of course we can.